sys/psa_crypto/Makefile.include

```diff
@@ -1,5 +1,6 @@
 # include the PSA headers
 INCLUDES += -I$(RIOTBASE)/sys/psa_crypto/include
+INCLUDES += -I$(RIOTBASE)/sys/include/psa_crypto
```

Comment:
I know I recommended this, if other people have a better way then please suggest.

Maybe it would be good to put a comment like:
```
# Workaround for mbedtls namespacing issue
# sys/include will include the conflicting header files in psa but requires a psa/*.h structure...

Reply:
Yep, good idea.
Also I had to change the header definitions to `PSA_CRYPTO_PSA_CRYPTO_H` etc. which is not so nice. Is there a way to not do that? (Building it works, but the static tests fail)

Reply:
I don't know about changing the name of the header guards... You could alternatively add the headers directly into the module `sys/psa_crypto/include/psa/` rather than in `sys/include/psa_crypto/psa/`... I don't know if that would break the convention of "public" headers vs internal though.

The real question is will this be hard to change in the future.  If we just go with something and someone comes along with an "uhhh actually" moment will that involve deprectations everywhere.
As long as you are not changing code I think this will be fine.

Reply:
Also we should note that the only reason for this has nothing to do with the psa work but the mbedtls namespacing...

If that gets corrected in an update or patch then we can revert to the standard way of doing it.

Reply:
Added a comment